src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/messages.js

```diff
@@ -56,5 +56,22 @@ const messages = defineMessages({
     defaultMessage: 'is not selected',
     description: 'Bold & underlined text for feedback if option is not selected',
   },
+
+  addAnswerRangeButtonText: {
+    id: 'authoring.answerwidget.answer.addAnswerRangeButton',
+    defaultMessage: 'Add Answer Range',
```

Comment:
nit: change this to 'Add answer range' to match the 'Add answer' option
<img width="324" alt="image" src="https://user-images.githubusercontent.com/56318341/228666318-c44aeb6a-6671-41bd-bf73-4fda733a62e8.png">